_sitegen/controllers/COCustomer.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+/**
+ * Controller for the first step of the cart checkout process, which is to ask the customer to login, register, or
+ * checkout anonymously.
+ *
+ * @module controllers/COCustomer
+ */
+
+/* API Includes */
+var Transaction = require('dw/system/Transaction');
+var URLUtils = require('dw/web/URLUtils');
+
+/* Script Modules */
+var app = require('~/cartridge/scripts/app');
+var guard = require('~/cartridge/scripts/guard');
+
+var Cart = require('~/cartridge/scripts/models/CartModel');
+var Content = require('~/cartridge/scripts/models/ContentModel');
+
+/**
+ * First step of the checkout is to choose the checkout type: returning, guest or create account checkout.
+ * Prepares the checkout initially: removes all payment instruments from the basket and clears all
+ * forms used in the checkout process, when the customer enters the checkout. The single steps (shipping, billing etc.)
+ * may not contain the form clearing, in order to support navigating forth and back in the checkout steps without losing
+ * already entered form values.
+ */
+function start() {
+    var oauthLoginForm = app.getForm('oauthlogin');
+    app.getForm('singleshipping').clear();
+    app.getForm('multishipping').clear();
+    app.getForm('billing').clear();
+
+    Transaction.wrap(function () {
+        Cart.goc().removeAllPaymentInstruments();
+    });
+
+
+    /* Klaviyo Started Checkout event tracking */
+    var KLCheckoutHelpers = require('*/cartridge/scripts/klaviyo/checkoutHelpers');
+    var customerEmail = KLCheckoutHelpers.getEmailFromBasket();
+    var KLTplVars = KLCheckoutHelpers.startedCheckoutHelper(true, customerEmail);
```

Comment:
@aidrian-maze is there a reason that for SG the var name for template variables is `KLTplVars` and for SFRA it is `templateVars`? If not can we use `templateVars` for both for consistency?